src/utils/hashPass.util.ts

```diff
@@ -0,0 +1,5 @@
+import { hash } from 'bcrypt';
+
+export const hashPassword = async (password: string) => {
+  return await hash(password, 14);
+};
```

Comment:
password will be automatically hashed when you use UserModel

I have written a pre hook that automatically hash passwords before creating new document in database